test/_utils.js

```diff
@@ -1,10 +1,26 @@
 /*
 * Test utils
 */
-
 const platform = process.platform;
 const arch = process.arch;
 
+/**
+ * Sets or unsets an environment variable and the corresponding npm_config variant.
+ *
+ * @param {string} name The UPPER_CASE name of the environment variable.
+ * @param {string} value The value to set - if falsy will be deleted.
+ */
+function setEnvVar(name, value) {
```

Comment:
This one I don't like very much. I prefer functions to perform only single action instead of being multi-purpose and performing different tasks according to theirs params. I would prefer it to be split into 2 params.

In addition, there's a possible bug: imagine `0` or `false` passed as `value`. In this case it will just delete env vars instead of setting theirs values.

Reply:
Oh, I just noticed that this is test-only but still, for readability, I would prefer to have `set` and `delete`. Like this, when you read test's code you can see that intention was actually to delete value and not just set it empty or something else.

Reply:
Yes I think you are right about that.